torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -523,6 +523,18 @@ struct Environment {
     return getSugaredVar(ident, range)->asValue(range, method);
   }
 
+  bool removeVar(const std::string& ident) {
```

Comment:
Why does this return a boolean ?

Reply:
The idea is to return `true` if there was a variable removed, `false` if not.